frontend/src/components/Events/EventComments.js

```diff
@@ -30,12 +33,15 @@ const Button1 = styled(Button) ({
 function EventComments(props) {
 
     const submitComment = () => {
-        props.addComment(props.userId, props.event, props.text);
+        axios.patch('http://localhost:3001/events/comment', {_id: props.event._id, comment: {user: firebase.auth().currentUser.email, text: props.text}}).then(() => {
+            props.addComment(firebase.auth().currentUser.email, props.event, props.text, props.events);
+        }).catch((err) => console.log(err))
```

Comment:
I think this needs code formatting. You should set up the Prettier VS Code extension, it'll save you a lot of time and effort.